<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TLMMainWindow</class>
 <widget class="QMainWindow" name="TLMMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TLM Plugin Manager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Leva/src/res.qrc">
    <normaloff>:/images/Icon_16.ico</normaloff>:/images/Icon_16.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Plugins on server</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTreeWidget" name="trvServerPlugins">
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="indentation">
            <number>0</number>
           </property>
           <column>
            <property name="text">
             <string>Plugin Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Description</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Version</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Release Date</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Connected Clients</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTreeWidget" name="trvClients">
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="indentation">
            <number>0</number>
           </property>
           <column>
            <property name="text">
             <string>Client Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Plugin Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Client Version</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Client Version Release Date</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Server Version</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Server Version Release Date</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Status</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QTextBrowser" name="textBrowser"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>513</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../Leva/src/res.qrc"/>
 </resources>
 <connections/>
</ui>
